controllers/controllers/services/instances/managed/controller_test.go

```diff
@@ -499,16 +499,75 @@ var _ = Describe("CFServiceInstance", func() {
 	})
 
 	When("the instance is deleted", func() {
+		BeforeEach(func() {
+			brokerClient.DeprovisionReturns(osbapi.ServiceInstanceOperationResponse{
+				Operation: "deprovision-op",
+			}, nil)
+			brokerClient.GetServiceInstanceLastOperationReturns(osbapi.LastOperationResponse{
```

Comment:
stubbing the last operation should not be necessary as the fainalizer does not check it anymore